requirements.txt

```diff
@@ -50,3 +50,4 @@ wandb>=0.11.1,<0.13.4
 ansible_vault>=2.1
 substrate-interface==1.2.4
 markupsafe==2.0.1
+subtensorapi>=0.1.0; sys_platform != "win32"
```

Comment:
let's use == for requirements, new library version integrated should trigger a new PR for this repository